apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-hosts/facility-hosts.component.ts

```diff
@@ -40,18 +38,18 @@ export class FacilityHostsComponent implements OnInit {
   routeAuth: boolean;
 
   ngOnInit(): void {
-    this.route.parent.params.subscribe(parentParams => {
-      this.facilityId = parentParams['facilityId'];
-      this.facilitiesManager.getFacilityById(this.facilityId).subscribe(facility => {
-        this.facility = facility;
-        this.refreshTable();
-      });
+    this.loading = true;
+    this.facility = this.entityStorageService.getEntity();
+    this.setAuthRights();
+    this.facilitiesManager.getFacilityById(this.facility.id).subscribe(facility => {
+      this.facility = facility;
+      this.refreshTable();
```

Comment:
If you add also facility name to storage in `facility-detail-page.component.ts`, you will be able to skip this call.

Reply:
Same as before.